Content.Server/Voting/Managers/VoteManager.DefaultVotes.cs

```diff
@@ -77,7 +77,7 @@ private void CreateRestartVote(IPlayerSession? initiator)
                 {
                     _chatManager.DispatchServerAnnouncement(Loc.GetString("ui-vote-restart-succeeded"));
                     var roundEnd = _entityManager.EntitySysManager.GetEntitySystem<RoundEndSystem>();
-                    roundEnd.EndRound();
+                    roundEnd.RequestRoundEnd(null, true, true);
```

Comment:
Maybe a separate vote for shuttle call, still keeping restart as a faster option if wanted